packages/letters/letters.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Client library for sending email over SMTP"
+description: "Simple to use SMTP client implementation for ocaml"
+maintainer: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+authors: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+homepage: "https://github.com/oxidizing/letters/"
+bug-reports: "https://github.com/oxidizing/letters/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
+  "mrmime" {= "0.3.0"}
+  "colombe" {= "0.3.0"}
+  "sendmail-lwt" {= "0.3.0"}
+  "fmt" {= "0.8.8"}
+  "x509" {>= "0.9.0"}
+  "ptime" {>= "0.8.5"}
+  "lwt" {>= "5.2.0" & < "6.0.0"}
```

Comment:
Are you sure this is going to break with lwt 6.0? Could you remove the upper-bound constraint if you are not sure? Major versions of Lwt tends to be well tested.

Reply:
Fixing with next time we push.

Reply:
Fixed.